src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf

```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81">
                 <source>This is not a valid Business Identifier Code (BIC).</source>
-                <target>Ovo nije validan poslovni identifikacijski broj (BIC).</target>
+                <target state="needs-review-translation">Ova vrijednost nije valjani Poslovni identifikacijski kod (BIC).</target>
```

Comment:
Same here:
```suggestion
                <target state="needs-review-translation">Ova vrijednost nije valjani poslovni identifikacijski kod (BIC).</target>
```

Reply:
I assumed other translations are correct in "hr" and removed the attribute